### ISSUE TITLE
quiver frontend - additional (external) info

### ISSUE THREAD

**Issue by MareenGeestmann** [OCR-D/quiver-frontend#53]
add additional (external) information

- [ ] link to gt workspace
- [ ] link to processor github repos
- [ ] link to workflow scripts
- [ ] processor versions on hover

**Comment by jfrer** [comment]
Currently this is not possible to implement:

> link to gt workspace

`metadata.gt_workspace.id` of each run leads to non-existing files, e.g.: https://github.com/OCR-D/quiver-data/blob/main/calvi_beutelschneider01_1627.ocrd.zip

> link to processor github repos

As far as I know, this data is currently missing in the API.

> link to workflow scripts

`metadata.ocr_workflow` of each run also leads to non-existing files, see also OCR-D/quiver-benchmarks#25.

> processor versions on hover

As far as I know, this data is also currently missing in the API.

**Comment by jfrer** [comment]
> > link to processor github repos
> 
> As far as I know, this data is currently missing in the API.

Actually it seems like this already has been done for the old (?) List-View in #31, so I try to replicate it for the Timeline-View.

**Comment by jfrer** [comment]
>> processor versions on hover
>
>As far as I know, this data is also currently missing in the API.

I found a way to map this data from the API where I got the processor github repo links from. 

But I'm not sure where this hover effect should happen. 

I feel like having this popup on hovering directly over the steps in the workflow view (eg: TR) might be misleading because we already have a hover effect there to let the user know they can click on the steps to open a more detailed view. 

The same could be said if this were implemented as a popup on hovering the link in the more detailed view. This could distract the user from seeing that they can click on this link. 

I would suggest to add the version somewhere in the detailed view, maybe in the header or directly below the header and above the table.

**Comment by MareenGeestmann** [comment]
> I would suggest to add the version somewhere in the detailed view, maybe in the header or directly below the header and above the table.

You're right, I agree, we should show it in the detail view between header and table.
